src/sentry/models/group.py

```diff
@@ -262,10 +262,11 @@ def save(self, *args, **kwargs):
         )
         super(Group, self).save(*args, **kwargs)
 
-    def get_absolute_url(self):
-        return absolute_uri(
-            reverse('sentry-group', args=[self.organization.slug, self.project.slug, self.id])
-        )
+    def get_absolute_url(self, referrer=None):
```

Comment:
Should this maybe come in as kwargs for extra query string parameters?

Reply:
sure can do